test/lib/next-test-utils.js

```diff
@@ -238,6 +254,13 @@ export function buildTS(args = [], cwd, env = {}) {
 export async function killApp(instance) {
   await new Promise((resolve, reject) => {
     treeKill(instance.pid, err => {
+      if (instance.dir) {
+        // since we force kill the dev processes we need to clean up the lock
+        // file so it doesn't break following tests
+        try {
+          rmdirSync(path.join(instance.dir, '.next.lock'))
```

Comment:
We need to fix this. Can the bootup wait the cooldown period and retry?

Reply:
Seems `SIGINT` isn't sent to the child process correctly for our tests on Windows 🤔